app/views/collection_drafts/show.html.erb

```diff
@@ -1,6 +1,7 @@
 <% content_for :header_title do %>
   <h2><%= display_entry_id(get_resource.draft, resource_name) %></h2>
   <p class="subtitle"><%= display_collection_entry_title(get_resource.draft) %></p>
+  <%= nrt_badge get_resource.draft %>
```

Comment:
also please use parentheses with this helper to be consistent. as you can see, the view helpers above are called with parentheses.